2023-12-28T21:09:26.427Z 2023-12-28T21:09:26.427Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:26.451Z INIT_REPORT Init Duration: 230.46 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:10.904Z 2023-12-28T21:11:10.904Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:10.985Z INIT_REPORT Init Duration: 750.93 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:10.986Z START RequestId: a79e8f64-2918-5c6d-b82f-d163cfb4608f Version: $LATEST

2023-12-28T21:11:10.986Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:11:10.986Z END RequestId: a79e8f64-2918-5c6d-b82f-d163cfb4608f

2023-12-28T21:11:10.986Z REPORT RequestId: a79e8f64-2918-5c6d-b82f-d163cfb4608f	Duration: 751.90 ms	Billed Duration: 752 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	
XRAY TraceId: 1-658de46e-52a4903e7c940cc8b8eb2b62	SegmentId: 1526afd20a4f5de9	Sampled: true	

2023-12-28T21:11:13.509Z 2023-12-28T21:11:13.509Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:13.645Z INIT_REPORT Init Duration: 772.26 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:13.645Z START RequestId: bff45e2e-cbcc-5cf7-bfbc-00292d7c5015 Version: $LATEST

2023-12-28T21:11:13.646Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:11:13.646Z END RequestId: bff45e2e-cbcc-5cf7-bfbc-00292d7c5015

2023-12-28T21:11:13.646Z REPORT RequestId: bff45e2e-cbcc-5cf7-bfbc-00292d7c5015	Duration: 773.15 ms	Billed Duration: 774 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:14:27.243Z 2023-12-28T21:14:27.243Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:27.363Z INIT_REPORT Init Duration: 778.87 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:14:27.363Z START RequestId: b520b937-2487-5bcb-8fff-e730c3f6af8d Version: $LATEST

2023-12-28T21:14:27.364Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:14:27.364Z END RequestId: b520b937-2487-5bcb-8fff-e730c3f6af8d

2023-12-28T21:14:27.364Z REPORT RequestId: b520b937-2487-5bcb-8fff-e730c3f6af8d	Duration: 779.68 ms	Billed Duration: 780 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:19:28.859Z 2023-12-28T21:19:28.859Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:19:28.979Z INIT_REPORT Init Duration: 784.49 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:19:28.980Z START RequestId: cd2f8fdd-94f2-547c-9c31-d1da7c6d9080 Version: $LATEST

2023-12-28T21:19:28.980Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:19:28.980Z END RequestId: cd2f8fdd-94f2-547c-9c31-d1da7c6d9080

2023-12-28T21:19:28.980Z REPORT RequestId: cd2f8fdd-94f2-547c-9c31-d1da7c6d9080	Duration: 785.40 ms	Billed Duration: 786 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:19:29.939Z 2023-12-28T21:19:29.939Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:19:30.056Z INIT_REPORT Init Duration: 798.80 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:19:30.056Z START RequestId: 4ef8580f-5462-59a7-abfc-0608bcfdccb1 Version: $LATEST

2023-12-28T21:19:30.056Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:19:30.056Z END RequestId: 4ef8580f-5462-59a7-abfc-0608bcfdccb1

2023-12-28T21:19:30.056Z REPORT RequestId: 4ef8580f-5462-59a7-abfc-0608bcfdccb1	Duration: 799.53 ms	Billed Duration: 800 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	

2023-12-28T21:01:03.773Z 2023-12-28T21:01:03.773Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:03.867Z INIT_REPORT Init Duration: 767.02 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:03.868Z START RequestId: 2fcc47c2-6ea1-5b35-8476-19c58ae5ecfc Version: $LATEST

2023-12-28T21:01:03.868Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:01:03.868Z END RequestId: 2fcc47c2-6ea1-5b35-8476-19c58ae5ecfc

2023-12-28T21:01:03.868Z REPORT RequestId: 2fcc47c2-6ea1-5b35-8476-19c58ae5ecfc	Duration: 767.96 ms	Billed Duration: 768 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	
XRAY TraceId: 1-658de20e-fc5cdc77ef4cbf8c7351099a	SegmentId: 314e2c21355115ab	Sampled: true	

2023-12-28T21:06:09.593Z 2023-12-28T21:06:09.593Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:09.728Z INIT_REPORT Init Duration: 790.70 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:09.728Z START RequestId: 9a9d5561-fdd4-511d-ba8c-0b661854af64 Version: $LATEST

2023-12-28T21:06:09.729Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:06:09.729Z END RequestId: 9a9d5561-fdd4-511d-ba8c-0b661854af64

2023-12-28T21:06:09.729Z REPORT RequestId: 9a9d5561-fdd4-511d-ba8c-0b661854af64	Duration: 791.80 ms	Billed Duration: 792 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	
XRAY TraceId: 1-658de340-ab4edbca55bcddfdb0acd284	SegmentId: 4c7fed775adc72d8	Sampled: true	

2023-12-28T21:09:26.221Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:14:28.162Z 2023-12-28T21:14:28.162Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:28.279Z INIT_REPORT Init Duration: 791.79 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:14:28.279Z START RequestId: 89e947d1-727f-5177-b194-19d6950dd531 Version: $LATEST

2023-12-28T21:14:28.280Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:14:28.280Z END RequestId: 89e947d1-727f-5177-b194-19d6950dd531

2023-12-28T21:14:28.280Z REPORT RequestId: 89e947d1-727f-5177-b194-19d6950dd531	Duration: 792.51 ms	Billed Duration: 793 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:19:27.085Z 2023-12-28T21:19:27.085Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:19:27.236Z INIT_REPORT Init Duration: 793.85 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:19:27.236Z START RequestId: 1d675d85-e89d-594c-980b-03820003fc5a Version: $LATEST

2023-12-28T21:19:27.236Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:19:27.236Z END RequestId: 1d675d85-e89d-594c-980b-03820003fc5a

2023-12-28T21:19:27.236Z REPORT RequestId: 1d675d85-e89d-594c-980b-03820003fc5a	Duration: 794.65 ms	Billed Duration: 795 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:19:28.039Z 2023-12-28T21:19:28.039Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:19:28.136Z INIT_REPORT Init Duration: 792.36 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:19:28.136Z START RequestId: 41d89e3e-9c62-5394-bb71-c03974fce768 Version: $LATEST

2023-12-28T21:19:28.136Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:19:28.136Z END RequestId: 41d89e3e-9c62-5394-bb71-c03974fce768

2023-12-28T21:19:28.136Z REPORT RequestId: 41d89e3e-9c62-5394-bb71-c03974fce768	Duration: 793.16 ms	Billed Duration: 794 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:01:09.068Z 2023-12-28T21:01:09.068Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:09.168Z INIT_REPORT Init Duration: 758.63 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:09.168Z START RequestId: 049f7472-cc15-5930-8074-2a7528417389 Version: $LATEST

2023-12-28T21:01:09.168Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:01:09.168Z END RequestId: 049f7472-cc15-5930-8074-2a7528417389

2023-12-28T21:01:09.168Z REPORT RequestId: 049f7472-cc15-5930-8074-2a7528417389	Duration: 759.53 ms	Billed Duration: 760 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:01:11.236Z 2023-12-28T21:01:11.236Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:11.371Z INIT_REPORT Init Duration: 773.47 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:11.371Z START RequestId: 8113bea4-4205-53f5-8d40-e0a3a653343c Version: $LATEST

2023-12-28T21:01:11.372Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:01:11.372Z END RequestId: 8113bea4-4205-53f5-8d40-e0a3a653343c

2023-12-28T21:01:11.372Z REPORT RequestId: 8113bea4-4205-53f5-8d40-e0a3a653343c	Duration: 774.19 ms	Billed Duration: 775 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	

2023-12-28T21:06:12.223Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:06:12.427Z 2023-12-28T21:06:12.427Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:12.456Z INIT_REPORT Init Duration: 233.07 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:14:26.323Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:01:02.773Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:01:03.054Z 2023-12-28T21:01:03.054Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:03.082Z INIT_REPORT Init Duration: 308.47 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:12.386Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:01:12.589Z 2023-12-28T21:01:12.589Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:12.621Z INIT_REPORT Init Duration: 235.35 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:03.128Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:06:10.852Z 2023-12-28T21:06:10.852Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:10.964Z INIT_REPORT Init Duration: 738.00 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:10.965Z START RequestId: 298edaef-4d76-5f2b-a61d-84e8a8f41040 Version: $LATEST

2023-12-28T21:06:10.965Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:06:10.965Z END RequestId: 298edaef-4d76-5f2b-a61d-84e8a8f41040

2023-12-28T21:06:10.965Z REPORT RequestId: 298edaef-4d76-5f2b-a61d-84e8a8f41040	Duration: 739.09 ms	Billed Duration: 740 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	
XRAY TraceId: 1-658de342-134b194b11114b911e1741c7	SegmentId: 31c17d2d70dce85e	Sampled: true	

2023-12-28T21:06:16.625Z 2023-12-28T21:06:16.625Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:16.728Z INIT_REPORT Init Duration: 756.28 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:16.728Z START RequestId: 709071e6-69c6-5d6f-92fc-2fe18c451204 Version: $LATEST

2023-12-28T21:06:16.729Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:06:16.729Z END RequestId: 709071e6-69c6-5d6f-92fc-2fe18c451204

2023-12-28T21:06:16.729Z REPORT RequestId: 709071e6-69c6-5d6f-92fc-2fe18c451204	Duration: 757.09 ms	Billed Duration: 758 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:09:28.131Z 2023-12-28T21:09:28.131Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:28.246Z INIT_REPORT Init Duration: 763.84 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:28.247Z START RequestId: 7ba74d55-f334-5487-b703-135215728865 Version: $LATEST

2023-12-28T21:09:28.263Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:09:28.263Z END RequestId: 7ba74d55-f334-5487-b703-135215728865

2023-12-28T21:09:28.263Z REPORT RequestId: 7ba74d55-f334-5487-b703-135215728865	Duration: 780.71 ms	Billed Duration: 781 ms	Memory Size: 512 MB	Max Memory Used: 42 MB	

2023-12-28T21:14:30.922Z 2023-12-28T21:14:30.922Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:31.003Z INIT_REPORT Init Duration: 737.19 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:14:31.003Z START RequestId: a232d21b-e6eb-54c7-ba4f-8c1c01982359 Version: $LATEST

2023-12-28T21:14:31.004Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:14:31.004Z END RequestId: a232d21b-e6eb-54c7-ba4f-8c1c01982359

2023-12-28T21:14:31.004Z REPORT RequestId: a232d21b-e6eb-54c7-ba4f-8c1c01982359	Duration: 737.93 ms	Billed Duration: 738 ms	Memory Size: 512 MB	Max Memory Used: 38 MB	

2023-12-28T21:19:26.193Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:01:12.276Z 2023-12-28T21:01:12.276Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:12.368Z INIT_REPORT Init Duration: 843.18 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:12.368Z START RequestId: fbcc7de0-57c2-5aa6-97c0-23e7b395201f Version: $LATEST

2023-12-28T21:01:12.369Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:01:12.369Z END RequestId: fbcc7de0-57c2-5aa6-97c0-23e7b395201f

2023-12-28T21:01:12.369Z REPORT RequestId: fbcc7de0-57c2-5aa6-97c0-23e7b395201f	Duration: 844.13 ms	Billed Duration: 845 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:06:03.047Z 2023-12-28T21:06:03.047Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:03.124Z INIT_REPORT Init Duration: 735.25 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:03.125Z START RequestId: 973bb1e6-d90e-5fba-b2c6-efe13c7e90d1 Version: $LATEST

2023-12-28T21:06:03.125Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:06:03.125Z END RequestId: 973bb1e6-d90e-5fba-b2c6-efe13c7e90d1

2023-12-28T21:06:03.125Z REPORT RequestId: 973bb1e6-d90e-5fba-b2c6-efe13c7e90d1	Duration: 736.58 ms	Billed Duration: 737 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	
XRAY TraceId: 1-658de33a-d28ac4d327e086840a61f8af	SegmentId: 2a3ccd056d15de95	Sampled: true	

2023-12-28T21:06:03.143Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:09:26.104Z 2023-12-28T21:09:26.104Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:26.186Z INIT_REPORT Init Duration: 795.49 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:26.186Z START RequestId: 9f41241f-9668-5632-91ee-7a18705bacf9 Version: $LATEST

2023-12-28T21:09:26.187Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:09:26.187Z END RequestId: 9f41241f-9668-5632-91ee-7a18705bacf9

2023-12-28T21:09:26.187Z REPORT RequestId: 9f41241f-9668-5632-91ee-7a18705bacf9	Duration: 796.25 ms	Billed Duration: 797 ms	Memory Size: 512 MB	Max Memory Used: 37 MB	

2023-12-28T21:11:03.328Z 2023-12-28T21:11:03.328Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:03.351Z INIT_REPORT Init Duration: 222.78 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:09.570Z 2023-12-28T21:11:09.570Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:09.665Z INIT_REPORT Init Duration: 729.06 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:09.665Z START RequestId: 2ea116a3-4e8c-5745-92cd-48a4a820c303 Version: $LATEST

2023-12-28T21:11:09.666Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:11:09.666Z END RequestId: 2ea116a3-4e8c-5745-92cd-48a4a820c303

2023-12-28T21:11:09.666Z REPORT RequestId: 2ea116a3-4e8c-5745-92cd-48a4a820c303	Duration: 730.11 ms	Billed Duration: 731 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	
XRAY TraceId: 1-658de46c-8d69c605c7d9b4b76e94e21a	SegmentId: 6009349a681cd560	Sampled: true	

2023-12-28T21:11:12.405Z 2023-12-28T21:11:12.405Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:12.505Z INIT_REPORT Init Duration: 798.73 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:12.505Z START RequestId: 8838451d-f514-5468-9f3f-1b1a13bbf88a Version: $LATEST

2023-12-28T21:11:12.506Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:11:12.506Z END RequestId: 8838451d-f514-5468-9f3f-1b1a13bbf88a

2023-12-28T21:11:12.506Z REPORT RequestId: 8838451d-f514-5468-9f3f-1b1a13bbf88a	Duration: 799.59 ms	Billed Duration: 800 ms	Memory Size: 512 MB	Max Memory Used: 43 MB	

2023-12-28T21:14:26.202Z 2023-12-28T21:14:26.202Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:26.303Z INIT_REPORT Init Duration: 763.75 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:14:26.303Z START RequestId: 412f21c5-894f-59a8-8c6c-7fc26e24cdee Version: $LATEST

2023-12-28T21:14:26.303Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:14:26.303Z END RequestId: 412f21c5-894f-59a8-8c6c-7fc26e24cdee

2023-12-28T21:14:26.304Z REPORT RequestId: 412f21c5-894f-59a8-8c6c-7fc26e24cdee	Duration: 764.51 ms	Billed Duration: 765 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:14:26.545Z 2023-12-28T21:14:26.545Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:26.568Z INIT_REPORT Init Duration: 244.55 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:19:26.079Z 2023-12-28T21:19:26.079Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:19:26.160Z INIT_REPORT Init Duration: 746.03 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:19:26.160Z START RequestId: 2a13a3f9-1c5d-5413-b552-fdfef4d09105 Version: $LATEST

2023-12-28T21:19:26.160Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:19:26.160Z END RequestId: 2a13a3f9-1c5d-5413-b552-fdfef4d09105

2023-12-28T21:19:26.160Z REPORT RequestId: 2a13a3f9-1c5d-5413-b552-fdfef4d09105	Duration: 746.72 ms	Billed Duration: 747 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:19:26.399Z 2023-12-28T21:19:26.399Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:19:26.425Z INIT_REPORT Init Duration: 231.87 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:09.186Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:01:09.390Z 2023-12-28T21:01:09.390Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:09.415Z INIT_REPORT Init Duration: 229.39 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:30.983Z 2023-12-28T21:09:30.983Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:31.062Z INIT_REPORT Init Duration: 794.26 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:31.062Z START RequestId: 9f88ea77-02f9-57f1-94b2-b470e4be1d3a Version: $LATEST

2023-12-28T21:09:31.064Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:09:31.064Z END RequestId: 9f88ea77-02f9-57f1-94b2-b470e4be1d3a

2023-12-28T21:09:31.064Z REPORT RequestId: 9f88ea77-02f9-57f1-94b2-b470e4be1d3a	Duration: 795.72 ms	Billed Duration: 796 ms	Memory Size: 512 MB	Max Memory Used: 40 MB	

2023-12-28T21:11:03.023Z 2023-12-28T21:11:03.023Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:03.105Z INIT_REPORT Init Duration: 736.97 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:03.105Z START RequestId: 7cb4ecfa-4094-5a5b-ad21-b2fda60dd6b9 Version: $LATEST

2023-12-28T21:11:03.106Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:11:03.106Z END RequestId: 7cb4ecfa-4094-5a5b-ad21-b2fda60dd6b9

2023-12-28T21:11:03.106Z REPORT RequestId: 7cb4ecfa-4094-5a5b-ad21-b2fda60dd6b9	Duration: 738.03 ms	Billed Duration: 739 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	
XRAY TraceId: 1-658de466-972366a0684e78e519281f8c	SegmentId: 7dade25237a2eabd	Sampled: true	

2023-12-28T21:19:30.904Z 2023-12-28T21:19:30.904Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:19:31.039Z INIT_REPORT Init Duration: 771.32 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:19:31.039Z START RequestId: 45ed8da2-15bb-5bcb-86b1-d3a5fd1bcf37 Version: $LATEST

2023-12-28T21:19:31.040Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:19:31.040Z END RequestId: 45ed8da2-15bb-5bcb-86b1-d3a5fd1bcf37

2023-12-28T21:19:31.040Z REPORT RequestId: 45ed8da2-15bb-5bcb-86b1-d3a5fd1bcf37	Duration: 772.35 ms	Billed Duration: 773 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:06:03.355Z 2023-12-28T21:06:03.355Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:03.376Z INIT_REPORT Init Duration: 233.16 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:12.107Z 2023-12-28T21:06:12.107Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:12.189Z INIT_REPORT Init Duration: 726.08 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:12.189Z START RequestId: 22de731b-eb9a-5604-9983-2d1c0e19687d Version: $LATEST

2023-12-28T21:06:12.205Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:06:12.205Z END RequestId: 22de731b-eb9a-5604-9983-2d1c0e19687d

2023-12-28T21:06:12.205Z REPORT RequestId: 22de731b-eb9a-5604-9983-2d1c0e19687d	Duration: 742.40 ms	Billed Duration: 743 ms	Memory Size: 512 MB	Max Memory Used: 40 MB	

